### ISSUE TITLE
How to disabled tab expand abbreviation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#248]
```
Is there a way to disable tab expand abbreviation for Dreamwaver
cs5 ?

Used to press tab for code indentation, with zencoding the whole bunch
of highlighted code get deleted when tab is pressed.

Thanks


```

---

Original issue reported on code.google.com by `sohwh...@gmail.com` on 4 Mar 2011 at 12:12

**Comment by GoogleCodeExporter** [comment]
```
Please be more descriptive.

Latest Zen Coding extension (0.7.2) extends abbreviations only if a strict set 
of conditions met. So, it'll never expand if selection present. Tab key 
replacing selection is a default editor behavior.

Finally, you can modify source of the extension code or menus.xml to remove Tab 
shortcut.
```

---

Original comment by `GreLIm...@gmail.com` on 7 Mar 2011 at 7:43

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 13 Mar 2011 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
In Dreamweaver CS5 for Windows 7 replace the shortcut "Tab" to 
"Ctrl+Alt+Shift+Space" in the file Menus.xml

1. Open the C:\Users\YourUserName\AppData\Roaming\Adobe\Dreamweaver 
CS5\en_US\Configuration\Menus\Menus.xml

2. Search for "Tab" (with quotes) 

3. Replace "Tab" to "Ctrl+Alt+Shift+Space "

4. Save the file

5. Restart Dreamweaver

Hope that helps,
Hugs,

Jonathan 'Santhus'

- - - - - - - - - - 

The line is replaced with:
<shortcut key="Ctrl+Alt+Shift+Space" command="dw.runCommand('ZenCoding.html', 
'expand_abbreviation_with_tab')" name="Expand Abbreviation with Zen Coding" 
id="DWMenu_Shortcut_ZenCoding_expandAbbreviationWithTab" />
```

---

Original comment by `jonathan...@gtempaccount.com` on 17 Mar 2011 at 9:31

**Comment by GoogleCodeExporter** [comment]
```
I'm having a similar issue in eclispe, it began by quite rudely overriding lots 
of the default controls for eclipse such as CTRL+D for delete line (not 
impressed!), most of which I've managed to remove so just CTRL+E activates it 
EXCEPT for the tab key

Whenever I'm typing and try to insert a tab after a variable e.g.

$someVar<press tab>

it gets translated to:

<1somevar></1somevar>

In the eclipse key editor ive filtered for zen coding and unbound all of its 
keys except ctrl+e yet its still doing it!


```

---

Original comment by `mpik...@googlemail.com` on 5 May 2011 at 10:10

**Comment by GoogleCodeExporter** [comment]
```
For those interested I figure out how to disable tab expand abbreviation for DW 
CS5.

For XP: 
C:\Documents and Settings\yr_username\Application Data\Adobe\Dreamweaver 
CS5\en_US\Configuration\Menus\Menus.xml

Open up Menus.xml, scroll to line 120 and comment out <shortcut key="Tab" and 
done.
```

---

Original comment by `sohwh...@gmail.com` on 5 May 2011 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
Sorry, I didn't really knew about this feature before these requests. The issue 
is fixed in version 0.7.5.
```

---

Original comment by `GreLIm...@gmail.com` on 21 May 2011 at 7:04
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
thanks is always good to share with the community to make zen coding better.
```

---

Original comment by `sohwh...@gmail.com` on 22 May 2011 at 1:51
